brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-2041 Fix: Block encounter an issue with react hook withSelect.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-2041
- Replaced withSelect with useSelect.
- Removed deprecated `__experimentalGetSettings` and used `getSettings` for Date.

### Screenshots

- https://share.getcloudapp.com/mXu2k8k6

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- For Column and container -
  - Drag and drop column.
  - Check variation is visible or not.
  - Add inner blocks, add a content, styling to block.
  - Verify it is visible on tablet, mobile in editor as well as frontend.
- For CF/GF and Forms
  - Drag and drop block.
  - Verify it is showing variation for block if available.
  - GF/CF Forms is rendering inside editor. Try creating new gf/cf forms and verify it is visible or not.
  - Add a child into forms blocks.
  - Try adding color to blocks.
  - Verify it is visible on tablet, mobile in editor as well as frontend.
- For Post block 
  - Drag and drop Grid/Masonry/Carousel/Post timeline.
  - Try changing post type and verify post are showing in block as per post type set.
  - Change Taxonomy type to categories, Tags and verify posts are showing accordingly.
  - Post Per Page, Columns and content setting.
  - Try adding colors to post.
- For Taxonomy
  - Drag and drop block.
  - Make sure you have different tags categories for post.
  - Verify according to tags and category taxonomy showing or not.
  - Try different layout, add a styling to block.
- For Table of content
  - Add a different heading on page and verify they are visible inside editor as well as on front or not.
  - Try different styling to TOC block.
- How-To
  - Drag and drop How-To
  - Edit the content.
  - Check schema is generated in JSON file by inspecting block.
  - Verify schema as per content added.


### Checklist:
